website/source/docs/auth/radius.html.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: "docs"
+page_title: "Auth Backend: RADIUS"
+sidebar_current: "docs-auth-radius"
+description: |-
+  The "radius" auth backend allows users to authenticate with Vault using an existing RADIUS server.
+---
+
+# Auth Backend: RADIUS
+
+Name: `radius`
+
+The "radius" auth backend allows users to authenticate with Vault using
+an existing RADIUS server that accepts the PAP authentication scheme. 
+
+The mapping of users to Vault policies is managed by using the
+`users/` path.
+
+Optionally, a configurable set of policies can be granted to all users that 
+can successfully authenticate but are not registered in the `users/` path.
+
+## Authentication
+
+The 'radius' auth backend accepts 
+
+#### Via the CLI
+
+```
+$ vault auth -method=userpass -path=radius \
+    username=foo \
+    password=bar
+```
+
+#### Via the API
+
+The endpoint for the login is `auth/radius/login/<username>`.
+
+The password should be sent in the POST body encoded as JSON.
+
+```shell
+$ curl $VAULT_ADDR/v1/auth/radius/login/mitchellh \
+    -d '{ "password": "foo" }'
+```
+
+The response will be in JSON. For example:
+
+```javascript
+{
+  "lease_id": "",
+  "renewable": false,
+  "lease_duration": 0,
+  "data": null,
+  "auth": {
+    "client_token": "c4f280f6-fdb2-18eb-89d3-589e2e834cdb",
+    "policies": [
+      "admins"
+    ],
+    "metadata": {
+      "username": "mitchellh"
+    },
+    "lease_duration": 0,
+    "renewable": false
+  }
+}
+```
+
+## Configuration
+
+First, you must enable the RADIUS auth backend:
+
+```
+$ vault auth-enable radius
+Successfully enabled 'radius' at 'radius'!
+```
+
+Now when you run `vault auth -methods`, the RADIUS backend is
+available:
+
+```
+Path       Type      Description
+token/     token     token based credentials
+radius/    radius
+```
+
+To use the ldap auth backend, it must first be configured with connection
+details for your RADIUS server.
+The configuration options are detailed below.
+
+Configuration is written to `auth/radius/config`.
+
+### Configuration parameters
```

Comment:
Can we not list out the arguments to endpoints here? The API docs is meant for just this.